lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons #:yell_at_patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+    #@yell_at_patrons =[]
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    @patrons.map(&:upcase) #found on google when I couldn't get.each to work
```

Comment:
Nice job finding another enumerable to use. The nice thing about `.map` is that it does change the original array so you do not need to use an accumulator with it.